tests/page_test.go

```diff
@@ -74,6 +74,18 @@ func TestPageEmulateMedia(t *testing.T) {
 	assert.True(t, res.ToBoolean(), "expected reduced motion setting to be 'reduce'")
 }
 
+func TestPageContent(t *testing.T) {
+	t.Parallel()
+
+	tb := newTestBrowser(t)
+	p := tb.NewPage(nil)
+
+	content := `<!DOCTYPE html><html><head></head><body><h1>Hello</h1></body></html>`
```

Comment:
Is there any reason for having `<!DOCTYPE html>` here?

Reply:
It's to test both branches of the `if` statement. See https://github.com/grafana/xk6-browser/pull/327#discussion_r884842527.